nc/source/config/config.ini

```diff
@@ -1,6 +1,11 @@
 [project]
 project_name=NcEngine Samples
 log_file_path=Diagnostics.log
+[memory]
+max_colliders=50000
```

Comment:
How do we fail if a user were to set this to a number such that we would run out of memory? ie (500,000,000,000)